nboard

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-
+# @JMW2020 5:15 PM CST 11/19/2020 Edit "Grammar and spelling fixes."
```

Comment:
no need to stick attribution into the file, its on the git history and on the contributors tab in github

Reply:
@xfnw Ok!